app/forms/DucrPartChiefChoice.scala

```diff
@@ -16,29 +16,15 @@
 
 package forms
 
-import forms.EnhancedMapping.requiredRadio
-import play.api.data.{Form, Forms, Mapping}
 import play.api.libs.json.{Json, OFormat}
-import utils.validators.forms.FieldValidator.isContainedIn
 
 case class DucrPartChiefChoice(choice: String) {
-  def isDucrPart = choice == DucrPartChiefChoice.IsDucrPart
+  def isDucrPart: Boolean = choice == DucrPartChiefChoice.IsDucrPart
 }
 
 object DucrPartChiefChoice {
```

Comment:
Is this class and object still needed?

Reply:
Yes, it is still part of the `Cache` case class, but I do not know if we can remove from there.